setup/install/providers/kubernetes-v2/gke/index.md

```diff
@@ -0,0 +1,61 @@
+---
+Layout: single
+title:  "Google Kubernetes Engine (GKE) Setup"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+This page describes how to set up a Kubernetes cluster on
+[GKE](https://cloud.google.com/kubernetes-engine/) that Spinnaker can
+manage and deploy workloads to. The process is very simple, but some
+considerations need to be made when configuring how Spinnaker will authenticate
+against your cluster.
+
+> Note: To manage and create clusters in a given project, you will need the
+> `roles.container/admin` role as described
+> [here](https://cloud.google.com/kubernetes-engine/docs/how-to/iam#predefined).
+
+# Create a cluster
+
+You can create a Kubernetes cluster on GKE using either
+[`gcloud`](https://cloud.google.com/sdk/gcloud/) or the [the
+Cloud Console](https://console.cloud.google.com/) as shown in the [official documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster#creating_a_cluster).
+Third party tools like
+[Terraform](https://www.terraform.io/docs/providers/google/r/container_cluster.html)
+work as well, and can be used to automate provisioning your clusters.
+
+# Download credentials
+
+Follow the instructions shown in [the official
+documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#generate_kubeconfig_entry)
+to download credentials.
+
+__Warning!__ Odds are, the credentials you've downloaded rely on calling
+`gcloud` to generate a token and authenticate against your cluster. This means
+that the user (e.g. `a.employee@example.org`) you've configured `gcloud` to
+authenticate as is making requests against the cluster, instead of a fixed
+[Kubernetes service
+account](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/).
```

Comment:
"Instead of" is a little ambiguous here.

Do you mean...
against the cluster instead of against a fixed service account
or do you mean...
the user I authenticate as instead of authenticating as a fixed service account?

Reply:
The latter